bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+---
+title: "Bitcoin Core 24.0"
+transcript_by: realdezzy via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=3UfrB7_ZOx0
+tags: ["bitcoin-core"]
+speakers: ["Sjors Provoost","Aaron Van Wirdum"]
+categories: ["podcast"]
+date: 2022-10-07
+---
+
+## Intro
+
+Aaron van Wirdum: 00:00:20
+
+Live from Utrecht, this is Bitcoin
+
+Sjors Provoost: 00:00:23
+
+Explained.
+
+Aaron van Wirdum: 00:00:24
+
+Hey Sjors.
+
+Sjors Provoost: 00:00:25
+
+What's up?
+
+Aaron van Wirdum: 00:00:26
+
+I'm good.
+
+Sjors Provoost: 00:00:27
+
+How do you like the weather?
+
+Aaron van Wirdum: 00:00:29
+
+Yeah, well, It was too hot all summer and then it was nice for about a week and now we're back to too cold.
```

Comment:
Removing "So..." form the end here is correct.

You could also remove the leading filler words. In general you should remove introductory filler words ('Like', 'So', 'Right') that start sentences but don’t contribute to the overall meaning
```suggestion
It was too hot all summer and then it was nice for about a week and now we're back to too cold.
```